src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -136,21 +139,21 @@ public final void testPathsFileWithLogVerification() throws IOException {
         antTask.execute();
 
         // then
-        final List<String> loggedMessages = antTask.getLoggedMessages();
+        final List<MessageLevelPair> loggedMessages = antTask.getLoggedMessages();
```

Comment:
Before this PR we had two Ant tasks. One in tests, another in Powermock tests.
Now we can reuse same task for all tests.